dashboard/app/controllers/lti_v1_controller.rb

```diff
@@ -196,6 +196,9 @@ def authenticate
         user = Services::Lti.initialize_lti_user(decoded_jwt)
         PartialRegistration.persist_attributes(session, user)
         session[:user_return_to] = destination_url
+        if DCDO.get('lti_account_linking_enabled', false)
+          redirect_to '/lti/v1/users/new' and return
```

Comment:
Please use router helper methods: https://guides.rubyonrails.org/routing.html#listing-existing-routes
```bash
➜ cd dashboard && rails routes -g lti/v1/users
```
```bash
      Prefix Verb URI Pattern                 Controller#Action
lti_v1_users GET  /lti/v1/users/new(.:format) lti/v1/users#new_user_landin
```
```suggestion
          redirect_to lti_v1_users_path
```